modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -51,14 +53,22 @@ occ config:app:set federation auto_accept_trusted --value '0'
 occ config:app:set federatedfilesharing auto_accept_trusted --value 'yes'
 ----
 
+== Working With Proxies
+
+There are ownCloud instances that are not connected to the internet. 
+They have no possibility to reach the public network. 
+Therefore Federation will not work without a proxy. 
+And then there are others that need to go through proxies which require authentication.
```

Comment:
ok, I think the first 3 statements will be sufficient.

Reply:
Updated.